crates/adapters/celestia/src/da_service/tests.rs

```diff
@@ -26,7 +26,13 @@ async fn collect_all_blobs_between(
     da_service: &CelestiaService,
     height_before: u64,
 ) -> anyhow::Result<(Vec<BlobWithSender>, Vec<BlobWithSender>)> {
-    let height_after = da_service.get_head_block_header().await?.height();
+    // Adding one more height to the current head to accommodate for blob inclusion.
+    // Even by the time submiPayForBlob has returned it should be included, we observed flakyness.
```

Comment:
```suggestion
    // Even though by the time submiPayForBlob has returned it should be included, we observed flakyness.
```
As long as this is the intended meaning